community: Update Korea University dataset URLs and titles

KU-MV opened the pull request:
Dear Nextstrain team,
I sent a comment on behalf of Professor Jin-Won Song from the Republic of Korea (ROK).
I am a researcher at Korea University in the ROK. 
I am pleased to inform you that our analysis for interested viruses using nextstrain workflow (system) is finished. 
We hope that our dataset is accessible via a front web page (Nextstrain community page).
We want to replace the database of the hantavirus, influenzavirus, SFTSV that was posted previously.
I provided the URL submitted to Github.
I would appreciate it if you could post our dataset on the front page (or community page) in the Nextstrain web page.
Please let me know the due date of the process for posting our datasets and give me a reply if you have any questions.

### Description of proposed changes
What is the goal of this pull request? What does this pull request change?

### Related issue(s)
<!-- Start typing the name of a related issue and GitHub will auto-suggest the issue number for you.  -->
Fixes #
Related to #

### Testing
What steps should be taken to test the changes you've proposed?
If you added or changed behavior in the codebase, did you update the tests, or do you need help with this?

<!-- 🙌 Thank you for contributing to Nextstrain! ✨ -->

victorlin commented on static-site/content/community-datasets.yaml:
```diff
@@ -200,3 +200,35 @@ data:
     maintainers: Center for Medical Genetics, Keio University School of Medicine
     date: '2022-03-23'
     title: Phylogenetic analysis of SARS-CoV-2 diversity in Japan
+  - url: https://nextstrain.org/community/KU-MolecularVirology/Hantavirus/L
+    maintainers: Jin-Won Song (Korea University)
+    date: '2022-04-23'
+    title: Hantavirus L segment
+  - url: https://nextstrain.org/community/KU-MolecularVirology/Hantavirus/M
+    maintainers: Jin-Won Song (Korea University)
+    date: '2022-04-23'
+    title: Hantavirus M segment
+  - url: https://nextstrain.org/community/KU-MolecularVirology/Hantavirus/S
+    maintainers: Jin-Won Song (Korea University)
+    date: '2022-04-23'
+    title: Hantavirus S segment
+  - url: https://nextstrain.org/community/KU-MolecularVirology/Influenzavirus/HA
+    maintainers: Man-Seong Park (Korea University)
+    date: '2022-04-23'
+    title: INFLUENZA HA
+  - url: https://nextstrain.org/community/KU-MolecularVirology/Influenzavirus/NA
+    maintainers: Man-Seong Park (Korea University)
+    date: '2022-04-23'
+    title: INFLUENZA NA
+  - url: https://nextstrain.org/community/KU-MolecularVirology/SFTSV/L
+    maintainers: Nam-Hyuk Cho (Seoul National University)
+    date: '2022-04-23'
+    title: Severe fever with thrombocytopenia syndrome virus (SFTSV) L segment
+  - url: https://nextstrain.org/community/KU-MolecularVirology/SFTSV/M
+    maintainers: Nam-Hyuk Cho (Seoul National University)
+    date: '2022-04-23'
+    title: Severe fever with thrombocytopenia syndrome virus (SFTSV) M segment
+  - url: https://nextstrain.org/community/KU-MolecularVirology/SFTSV/S
+    maintainers: Nam-Hyuk Cho (Seoul National University)
+    date: '2022-04-23'
+    title: Severe fever with thrombocytopenia syndrome virus (SFTSV) S segment
```
Hi @KU-MolecularVirology,

Thank you for sending your request via GitHub. Instead of adding new lines at the end of the file, could you replace these existing lines?

https://github.com/nextstrain/nextstrain.org/blob/ff13cb7bc72c35a94d0cf04a12f527a3ec75f28e/static-site/content/community-datasets.yaml#L59-L90

Otherwise, we have duplicate entries for your datasets:

![duplicate-community-datasets](https://user-images.githubusercontent.com/13424970/165147741-d1c007bf-e53b-44e5-8bf3-2019907c358d.png)

KU-MV replied:
Thanks your comment.  

I apologize for the confusion.

We want to replace URL from previous version to given URL.
We will re-upload  new URL of the dataset to change the URL from previous version (updated date 2022-03-03).

As soon as possible, we will update final repositories.

We really appreciate your great supports.

Sincerely,

KU-MV replied:
Dear Nextstrain team,
I sent a comment on behalf of Professor Jin-Won Song (Korea University) and Nam-Hyuk Cho (Seoul National University)from the Republic of Korea (ROK).

I am a researcher at Korea University in the ROK.

Initially, we contact Dr. Trevor Bedford, and we have prepared three community sites (about three different viruses; swine influenza virus, SFTSV, and hantavirus) that would be fit in the web of Nextstrain Community.

First dataset (git owner: modoree) was posted on Nextstrain community page previously. However, we thought that the URL is not clear for the description of virus characterization and contributed groups. Thus, we want to replace the URL (database of the hantavirus, influenzavirus, and SFTSV).

I provided the new URL submitted to Github.
1) Hantavirus
https://nextstrain.org/community/KU-MV/Hantavirus/L
https://nextstrain.org/community/KU-MV/Hantavirus/M
https://nextstrain.org/community/KU-MV/Hantavirus/S
2) Influenzavirus
https://nextstrain.org/community/KU-MV/Influenzavirus/HA 
https://nextstrain.org/community/KU-MV/Influenzavirus/NA
3) SFTSV
https://nextstrain.org/community/KU-MV/SFTSV/L
https://nextstrain.org/community/KU-MV/SFTSV/M
https://nextstrain.org/community/KU-MV/SFTSV/S

We really appreciate your great supports.  

As soon as, we will remove the previous URL (supported by modoree).
Would you please let us know what we have to do for this? 

I would appreciate it if you could replace the URL and post our dataset on the front page (or community page) on the Nextstrain web page.

Please let me know the due date of the process for posting our datasets and give me a reply if you have any questions.

Thank you so much for your kind consideration. We are looking forward to hearing from you soon.

Sincerely,

victorlin replied:
Hello @KU-MV,

I understand that you want to replace the `modoree` URLs to use your new GitHub user `KU-MV`. I have updated this Pull Request with the necessary changes: 4a1e4a1273a9c50e794e82d95739fbef125d0aab

Unfortunately, we are unable to provide a preview website for you to confirm the changes. However, here is a screenshot when I test it on my computer:

<img width="500" alt="KU-MV dataset listing preview" src="https://user-images.githubusercontent.com/13424970/165213888-0d40230a-ba25-44de-a805-2dcb8987e713.png">

Let me know if this looks good, and we can make these changes on [nextstrain.org/community](https://nextstrain.org/community/).

KU-MV commented:
Dear Nextstrain team,
I sent a comment on behalf of Professor Jin-Won Song (Korea University) and
Nam-Hyuk Cho (Seoul National University)from the Republic of Korea (ROK).

I am a researcher at Korea University in the ROK.

Initially, we contact Dr. Trevor Bedford, and we have prepared three
community sites (about three different viruses; swine influenza virus,
SFTSV, and hantavirus) that would be fit in the web of Nextstrain Community.

First dataset (git owner: modoree) was posted on Nextstrain community page
previously. However, we thought that the URL is not clear for the
description of virus characterization and contributed groups. Thus, we want
to replace the URL (database of the hantavirus, influenzavirus, and SFTSV).

I provided the new URL submitted to Github.

   1. Hantavirus
   https://nextstrain.org/community/KU-MV/Hantavirus/L
   https://nextstrain.org/community/KU-MV/Hantavirus/M
   https://nextstrain.org/community/KU-MV/Hantavirus/S
   2. Influenzavirus
   https://nextstrain.org/community/KU-MV/Influenzavirus/HA
   https://nextstrain.org/community/KU-MV/Influenzavirus/NA
   3. SFTSV
   https://nextstrain.org/community/KU-MV/SFTSV/L
   https://nextstrain.org/community/KU-MV/SFTSV/M
   https://nextstrain.org/community/KU-MV/SFTSV/S

We really appreciate your great supports.

As soon as, we will remove the previous URL (supported by modoree).
Would you please let us know what we have to do for this?

I would appreciate it if you could replace the URL and post our dataset on
the front page (or community page) on the Nextstrain web page.

Please let me know the due date of the process for posting our datasets and
give me a reply if you have any questions.

Thank you so much for your kind consideration. We are looking forward to
hearing from you soon.

Sincerely,

2022년 4월 26일 (화) 오전 3:08, Victor Lin ***@***.***>님이 작성:

> ***@***.**** requested changes on this pull request.
> ------------------------------
>
> In static-site/content/community-datasets.yaml
> <https://github.com/nextstrain/nextstrain.org/pull/520#discussion_r857895362>
> :
>
> > +  - url: https://nextstrain.org/community/KU-MolecularVirology/Hantavirus/L
> +    maintainers: Jin-Won Song (Korea University)
> +    date: '2022-04-23'
> +    title: Hantavirus L segment
> +  - url: https://nextstrain.org/community/KU-MolecularVirology/Hantavirus/M
> +    maintainers: Jin-Won Song (Korea University)
> +    date: '2022-04-23'
> +    title: Hantavirus M segment
> +  - url: https://nextstrain.org/community/KU-MolecularVirology/Hantavirus/S
> +    maintainers: Jin-Won Song (Korea University)
> +    date: '2022-04-23'
> +    title: Hantavirus S segment
> +  - url: https://nextstrain.org/community/KU-MolecularVirology/Influenzavirus/HA
> +    maintainers: Man-Seong Park (Korea University)
> +    date: '2022-04-23'
> +    title: INFLUENZA HA
> +  - url: https://nextstrain.org/community/KU-MolecularVirology/Influenzavirus/NA
> +    maintainers: Man-Seong Park (Korea University)
> +    date: '2022-04-23'
> +    title: INFLUENZA NA
> +  - url: https://nextstrain.org/community/KU-MolecularVirology/SFTSV/L
> +    maintainers: Nam-Hyuk Cho (Seoul National University)
> +    date: '2022-04-23'
> +    title: Severe fever with thrombocytopenia syndrome virus (SFTSV) L segment
> +  - url: https://nextstrain.org/community/KU-MolecularVirology/SFTSV/M
> +    maintainers: Nam-Hyuk Cho (Seoul National University)
> +    date: '2022-04-23'
> +    title: Severe fever with thrombocytopenia syndrome virus (SFTSV) M segment
> +  - url: https://nextstrain.org/community/KU-MolecularVirology/SFTSV/S
> +    maintainers: Nam-Hyuk Cho (Seoul National University)
> +    date: '2022-04-23'
> +    title: Severe fever with thrombocytopenia syndrome virus (SFTSV) S segment
>
> Hi @KU-MolecularVirology <https://github.com/KU-MolecularVirology>,
>
> Thank you for sending your request via GitHub. Instead of adding new lines
> at the end of the file, could you replace these existing lines?
>
>
> https://github.com/nextstrain/nextstrain.org/blob/ff13cb7bc72c35a94d0cf04a12f527a3ec75f28e/static-site/content/community-datasets.yaml#L59-L90
>
> Otherwise, we have duplicate entries for your datasets:
>
> [image: duplicate-community-datasets]
> <https://user-images.githubusercontent.com/13424970/165147741-d1c007bf-e53b-44e5-8bf3-2019907c358d.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/nextstrain/nextstrain.org/pull/520#pullrequestreview-952263095>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AY3KVF24E5DJMUSOEJCX5XTVG3NRNANCNFSM5UEMUQMQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Kyungmin Park

Department of Microbiology
College of Medicine, Korea University

E-mail:
*kmpark0131@ ***@***.***>korea.ac.kr <http://korea.ac.kr/>*
*kmpark0131@ ***@***.***>gmail.com <http://gmail.com/>*

Address:
#208, Institute of Biomedical Science & Food Safety
Korea University, Seoul, South Korea, 136-713

Office: 82-2-920-6353
Cell: 82-10-4892-0933

KU-MV reviewed (commented):
Thats great.

We really appreciate your great supports.

Sincerely,

KU-MV commented:
Dear, Victor Lin,

That's great. 

I would appreciate it if you could update the preview version referred from you and post it on [nextstrain.org/community].

We really appreciate your great support.

Sincerely,

KU-MV commented:
Dear, Victor Lin,

That's great.

I would appreciate it if you could update the preview version referred from
you and post it on [nextstrain.org/community].

We really appreciate your great support.

Sincerely,

2022년 4월 26일 (화) 오후 12:24, Victor Lin ***@***.***>님이 작성:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In static-site/content/community-datasets.yaml
> <https://github.com/nextstrain/nextstrain.org/pull/520#discussion_r858225183>
> :
>
> > +  - url: https://nextstrain.org/community/KU-MolecularVirology/Hantavirus/L
> +    maintainers: Jin-Won Song (Korea University)
> +    date: '2022-04-23'
> +    title: Hantavirus L segment
> +  - url: https://nextstrain.org/community/KU-MolecularVirology/Hantavirus/M
> +    maintainers: Jin-Won Song (Korea University)
> +    date: '2022-04-23'
> +    title: Hantavirus M segment
> +  - url: https://nextstrain.org/community/KU-MolecularVirology/Hantavirus/S
> +    maintainers: Jin-Won Song (Korea University)
> +    date: '2022-04-23'
> +    title: Hantavirus S segment
> +  - url: https://nextstrain.org/community/KU-MolecularVirology/Influenzavirus/HA
> +    maintainers: Man-Seong Park (Korea University)
> +    date: '2022-04-23'
> +    title: INFLUENZA HA
> +  - url: https://nextstrain.org/community/KU-MolecularVirology/Influenzavirus/NA
> +    maintainers: Man-Seong Park (Korea University)
> +    date: '2022-04-23'
> +    title: INFLUENZA NA
> +  - url: https://nextstrain.org/community/KU-MolecularVirology/SFTSV/L
> +    maintainers: Nam-Hyuk Cho (Seoul National University)
> +    date: '2022-04-23'
> +    title: Severe fever with thrombocytopenia syndrome virus (SFTSV) L segment
> +  - url: https://nextstrain.org/community/KU-MolecularVirology/SFTSV/M
> +    maintainers: Nam-Hyuk Cho (Seoul National University)
> +    date: '2022-04-23'
> +    title: Severe fever with thrombocytopenia syndrome virus (SFTSV) M segment
> +  - url: https://nextstrain.org/community/KU-MolecularVirology/SFTSV/S
> +    maintainers: Nam-Hyuk Cho (Seoul National University)
> +    date: '2022-04-23'
> +    title: Severe fever with thrombocytopenia syndrome virus (SFTSV) S segment
>
> Hello @KU-MV <https://github.com/KU-MV>,
>
> I understand that you want to replace the modoree URLs to use your new
> GitHub user KU-MV. I have updated this Pull Request with the necessary
> changes: 4a1e4a1
> <https://github.com/nextstrain/nextstrain.org/commit/4a1e4a1273a9c50e794e82d95739fbef125d0aab>
>
> Unfortunately, we are unable to provide a preview website for you to
> confirm the changes. However, here is a screenshot when I test it on my
> computer:
>
> [image: KU-MV dataset listing preview]
> <https://user-images.githubusercontent.com/13424970/165213888-0d40230a-ba25-44de-a805-2dcb8987e713.png>
>
> Let me know if this looks good, and we can make these changes on
> nextstrain.org/community.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/nextstrain/nextstrain.org/pull/520#discussion_r858225183>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AY3KVF226JAZPGREDRKX6BDVG5OVFANCNFSM5UEMUQMQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Kyungmin Park

Department of Microbiology
College of Medicine, Korea University

E-mail:
*kmpark0131@ ***@***.***>korea.ac.kr <http://korea.ac.kr/>*
*kmpark0131@ ***@***.***>gmail.com <http://gmail.com/>*

Address:
#208, Institute of Biomedical Science & Food Safety
Korea University, Seoul, South Korea, 136-713

Office: 82-2-920-6353
Cell: 82-10-4892-0933

victorlin reviewed (approved):
Looks good to me.

The PR check [failed](https://github.com/nextstrain/nextstrain.org/pull/520/checks?check_run_id=6168878075) but I pushed the same changes to an in-repo branch victorlin/test and the branch check [succeeded](https://github.com/nextstrain/nextstrain.org/pull/520/checks?check_run_id=6180603331). So the failure is likely due to Travis CI limiting environment variables for PRs from forks, not the file changes.

Requesting @nextstrain/core for additional review.

huddlej commented:
Thanks, @victorlin! The content of the PR looks good, although we should figure out internally why these unrelated tests fail for this kind of PR.

@KU-MV One small issue I noticed with the builds is that their `build_url` in the Auspice config JSON needs to be updated to point to the `KU-MV` repository paths instead of the `modoree` paths. For example, the Hantavirus L build (https://nextstrain.org/community/KU-MV/Hantavirus/L) still says "Built with [modoree/htv](https://github.com/modoree/htv)." at the top of the main window in Auspice.

joverlee521 reviewed (commented):
This is minor since we don't display the `date` in our UI yet, but `date` should reflect the date the underlying dataset was updated. 

This would be a good opportunity to confirm our SOP for adding community datasets. Should we run `scripts/parse-community-data.js` whenever we get update `community-datasets.yaml`?

KU-MV commented:
> Thanks, @victorlin! The content of the PR looks good, although we should figure out internally why these unrelated tests fail for this kind of PR.
> 
> @KU-MV One small issue I noticed with the builds is that their `build_url` in the Auspice config JSON needs to be updated to point to the `KU-MV` repository paths instead of the `modoree` paths. For example, the Hantavirus L build (https://nextstrain.org/community/KU-MV/Hantavirus/L) still says "Built with [modoree/htv](https://github.com/modoree/htv)." at the top of the main window in Auspice.

Thanks your reply. As soon as possible, we will correct build_url in the JSON file.

KU-MV commented:
> Thanks, @victorlin! The content of the PR looks good, although we should figure out internally why these unrelated tests fail for this kind of PR.
> 
> @KU-MV One small issue I noticed with the builds is that their `build_url` in the Auspice config JSON needs to be updated to point to the `KU-MV` repository paths instead of the `modoree` paths. For example, the Hantavirus L build (https://nextstrain.org/community/KU-MV/Hantavirus/L) still says "Built with [modoree/htv](https://github.com/modoree/htv)." at the top of the main window in Auspice.

Thanks your reply. As soon as possible, we will correct build_url in the JSON file.

KU-MV commented:
> Thanks, @victorlin! The content of the PR looks good, although we should figure out internally why these unrelated tests fail for this kind of PR.
> 
> @KU-MV One small issue I noticed with the builds is that their `build_url` in the Auspice config JSON needs to be updated to point to the `KU-MV` repository paths instead of the `modoree` paths. For example, the Hantavirus L build (https://nextstrain.org/community/KU-MV/Hantavirus/L) still says "Built with [modoree/htv](https://github.com/modoree/htv)." at the top of the main window in Auspice.

Dear, John Huddleston
Thank you for your point. Based on your suggestion, we corrected the build_url and updated date.
I committed and updated the corrected files to git.
Please let me know the extra works of the process for posting our datasets and give me a reply if you have any questions.

Sincerely,

KU-MV commented:
> Looks good to me.
> 
> The PR check [failed](https://github.com/nextstrain/nextstrain.org/pull/520/checks?check_run_id=6168878075) but I pushed the same changes to an in-repo branch [`victorlin/test`](https://github.com/nextstrain/nextstrain.org/tree/victorlin/test) and the branch check [succeeded](https://github.com/nextstrain/nextstrain.org/pull/520/checks?check_run_id=6180603331). So the failure is likely due to Travis CI limiting environment variables for PRs from forks, not the file changes.
> 
> Requesting @nextstrain/core for additional review.

Dear, Victor Lin
Thank you for your point. Based on the suggestion, we corrected the build_url and updated date.
I committed and updated the corrected files to git.
Please let me know the extra work of the process for posting our datasets and give me a reply if you have any questions.
I have a question, Do we continuously ask for the pull requests when the dataset was updated ?
Or the dataset will be synced with git in realtime ?
Sincerely,

KU-MV commented:
> Looks good to me.
> 
> The PR check [failed](https://github.com/nextstrain/nextstrain.org/pull/520/checks?check_run_id=6168878075) but I pushed the same changes to an in-repo branch [`victorlin/test`](https://github.com/nextstrain/nextstrain.org/tree/victorlin/test) and the branch check [succeeded](https://github.com/nextstrain/nextstrain.org/pull/520/checks?check_run_id=6180603331). So the failure is likely due to Travis CI limiting environment variables for PRs from forks, not the file changes.
> 
> Requesting @nextstrain/core for additional review.

Dear, Victor Lin
We removed the pathway by modoree at Apr. 27 th (South Korea).
Please update our URL by KU-MV pathway.
We really appreciate your great support.
Sincerely,

victorlin commented:
@joverlee521 good point, I've created a separate issue for us to discuss that: #522

victorlin commented:
Hello @KU-MV,

I see that you have updated the `build_url`. This looks good to me. You only need to make a new Pull Request if you need to change the URL, maintainers, or title of any dataset.

If you make a new git commit to your dataset repository like you did for KU-MV/Hantavirus@c0edb1e304dd9dbbd1665d40d3d3f843dd305b76, this will show up on nextstrain.org in realtime.

I will wait approval by another team member, then this can be merged.

victorlin commented:
@KU-MV this Pull Request has been merged but we need to manually update nextstrain.org. I will let you know once it happens.

victorlin commented:
@KU-MV These changes are now live on https://nextstrain.org/community/.
